tests/integration/Owncloud/OcisPhpSdk/DriveTest.php

```diff
@@ -68,4 +69,22 @@ public function testDeleteEnabledDrive(): void
         $this->expectExceptionMessage('invalidRequest - error: bad request: can\'t purge enabled space');
         $this->drive->delete();
     }
+
+    public function testGetDriveRole(): void
```

Comment:
Note: we need to add test case for testGetDriveRole (added in PR https://github.com/owncloud/ocis-php-sdk/pull/225 ) to test exception handling when running on "stable" ocis v5.

I added that to the to-do list of issue #223